core/src/test/java/com/exadel/aem/toolkit/core/injectors/models/ITestModel.java

```diff
@@ -0,0 +1,15 @@
+package com.exadel.aem.toolkit.core.injectors.models;
+
+import com.exadel.aem.toolkit.core.injectors.annotations.RequestSelectors;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+
+@Model(adaptables = SlingHttpServletRequest.class,
+    defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL
+)
+public interface ITestModel {
```

Comment:
(cosmetic) I would suggest renaming it to ITestModelSelectors (similar to ITestModelSuffix in #235 )
or moving the method to TestModelSelectors to have one model for all cases

Reply:
I thought that one interface would be enough to test how injector works with methods.